assembly/src/parsers/io_ops/mem_ops.rs

```diff
@@ -164,341 +138,385 @@ fn push_mem_addr(span_ops: &mut Vec<Operation>, op: &Token) -> Result<(), Assemb
 #[cfg(test)]
 mod tests {
     use super::{
-        super::{
-            parse_loadw, parse_pop, parse_popw, parse_push, parse_pushw, parse_storew,
-            tests::get_parsing_error, Felt,
-        },
+        super::{parse_mem_read, parse_mem_write, tests::get_parsing_error, Felt},
         AssemblyError, Operation, Token,
     };
 
     // TESTS FOR PUSHING VALUES ONTO THE STACK (PUSH)
     // ============================================================================================
 
     #[test]
-    fn push_mem() {
+    fn mem_load() {
```

Comment:
I really like the helper function for local memory operations. Should we do something similar for regular memory operations as well?